docs/setup-guide/arktos-with-mizar-cni.md

```diff
@@ -0,0 +1,40 @@
+# Arktos to Enforce the Multi-tenant with Mizar Network Feature
+
+This document captures the steps applied to an Arktos cluster lab enabling the unique multi-tenant network feature. The machines in this lab used are AWS EC2 t2-large (2 CPUs, 8GB mem), Ubuntu 18.04 LTS.
+
+The steps might change with the progress of development.
+  
+1. Update Kernel
```

Comment:
This sentence reads "Update Kernel To check kernel, run following command" . See: https://github.com/Click2Cloud-Centaurus/arktos/blob/default-cni-mizar/docs/setup-guide/arktos-with-mizar-cni.md
Please cleanup the documentation. This is public-facing documentation.

Reply:
Removed the unnecessary contents from the guide, Thanks.